src/lib/drupal/staffProfileSideNav.ts

```diff
@@ -0,0 +1,23 @@
+import { SidebarData } from '@/types/formatted/staffProfile'
+
+export const buildStaffProfileSidebarData = (
+  title: string,
+  fieldOfficePath: string
+): SidebarData => {
+  const formattedTitle = title.replace(/"/g, '&quot;')
```

Comment:
This is fine, but I wonder if it's even necessary now that we're rendering the title in a React element, which should handle it without this encoding, no problem. 🤷

Reply:
worth testing, i'll add this to https://github.com/department-of-veterans-affairs/va.gov-cms/issues/21160